cypress/integration/hosts/host_filtering.ts

```diff
@@ -156,98 +156,121 @@ describe("Hosts page filtering from URL", () => {
   });
 });
 
-describe("Hosts page filtering from table filters", () => {
-  before(() => {
-    cy.login();
-  });
+describe(
+  "Hosts page filtering from table filters",
+  { scrollBehavior: false },
+  () => {
+    before(() => {
+      cy.login();
+    });
 
-  beforeEach(() => {
-    cy.preserveCookies();
-    cy.visit(`${hostsRoute}?limit=100&page=0`);
-    cy.dataCy("hosts-table").should("be.visible");
-    cy.dataCy("hosts-table").should("not.have.attr", "data-loading", "true");
-  });
+    beforeEach(() => {
+      cy.viewport(1920, 1080);
+      cy.preserveCookies();
+      cy.setCookie("This is an important notification", "true");
+      cy.visit(`${hostsRoute}?limit=100&page=0`);
+      cy.dataCy("hosts-table").should("be.visible");
+      cy.dataCy("hosts-table").should("not.have.attr", "data-loading", "true");
+      cy.dataCy(distroFilterIconDataCy)
+        .should("be.visible")
+        .should("not.be.disabled");
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(1000);
+    });
 
-  it("Filters hosts with input value when Enter key is pressed", () => {
-    cy.dataCy(distroFilterIconDataCy).click();
+    it("Filters hosts with input value when Enter key is pressed", () => {
+      cy.dataCy(distroFilterIconDataCy)
+        .should("be.visible")
+        .should("not.be.disabled");
+      cy.dataCy(distroFilterIconDataCy).scrollIntoView();
+      cy.dataCy(distroFilterIconDataCy)
+        .should("be.visible")
+        .should("not.be.disabled");
+      cy.dataCy(distroFilterIconDataCy).click();
```

Comment:
Since these assertions are already checked in `beforeEach`, I think you might be able to simplify this whole block to `     cy.dataCy(distroFilterIconDataCy).scrollIntoView().click()` 

(I tested on this [patch](https://spruce.mongodb.com/version/62d1b0da3066155eda46f370/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC))